apps/analytics/src/scripts/7_participant_activity_performance.py

```
@@ -0,0 +1,61 @@
+# This script compute the participant activity performance analytics
+# ! This script is a copy of the corresponding notebook content and needs to be kept in sync with it
+
+
+import os
+import json
```

Ruff (0.8.2)

`json` imported but unused

Remove unused import: `json`

(F401)